packages/transactions/tests/builder.test.ts

```diff
@@ -1299,7 +1299,7 @@ test.skip('Make sponsored STX token transfer with sponsor fee estimate', async (
   const sponsorSignedTx = await sponsorTransaction(sponsorOptions);
 
   expect(fetchMock.mock.calls.length).toEqual(1);
-  expect(fetchMock.mock.calls[0][0]).toEqual(network.getTransferFeeEstimateApiUrl());
+  expect(fetchMock.mock.calls[0][0]).toEqual(network.getTransactionFeeEstimateApiUrl());
```

Comment:
Unclear whether test is wrong or implementation is wrong. It seems, there used to only be `transfer` but now there is `transfer` and `transaction` in the API urls.

Reply:
@zone117x do you, by any chance, know more about `Transfer` vs `Transaction`?

e.g. Is a transfer just a simple transaction (w/o smart contract calling)

Reply:
@janniks this PR and the linked issues should provide some context https://github.com/hirosystems/stacks.js/pull/1134

The RPC endpoint associated with `transaction` provides smart fee estimation, while the previous one associated with `transfer` was essentially a constant. More info in the openAPI docs for the endpoints: https://github.com/stacks-network/stacks-blockchain/blob/6c5349b0422c5c88db02d578720467c16b102ab2/docs/rpc/openapi.yaml#L283